schema/schema.js

```diff
@@ -24,7 +26,8 @@ const {
   GraphQLInt,
   GraphQLList,
   GraphQLNonNull,
-  GraphQLFloat
+  GraphQLFloat,
+  GraphQLInterfaceType
```

Comment:
I would like to see how this was used, only cause im curious.

Reply:
I didn't end up having to use the interface after all. I should probably remove that :+1: